packages/cli/src/commands/index.ts

```diff
@@ -345,6 +345,10 @@ export function registerCommands(program: Command) {
     .option('--lax', 'Do not require environment variables to be set')
     .option('--frontend', 'Only validate the frontend configuration')
     .option('--deprecated', 'Output deprecated configuration settings')
+    .option(
+      '--strict',
+      'Ensure that the provided config(s) has no errors and does not contain keys not in the schema.',
```

Comment:
I think it's worth a bit of time to try to describe this in a more concise way to help this fit without wrapping in terminals. For example:

> Enable strict config validation, forbidding errors and unknown keys

Drop the `.` 😁